client/src/components/survey/SurveyPage1.jsx

```diff
@@ -180,9 +192,19 @@ const SurveyPage1 = props => {
           </Grid>
         </Grid>
         <Grid item xs={12}>
-          <Button onClick={submitSurveyPage1} variant="outlined" color="secondary" className={classes.continueButton}>
-            CONTINUE
-          </Button>
+          {dailyfeeling <= 2.0 && dailySymptomsFeeling <= 2.0 && dailyComparedToYesterday <= 2.0 ? (
+            <PopUpContactAlert
+              handleClickOpen={handleClickOpen}
+              handleClose={handleClose}
+              submitSurveyPage1={submitSurveyPage1}
+              open={open}
```

Comment:
When passing state between components, it's best to use Redux. Although, I don't think many of these props need to be passed at all as mentioned in other comments.